src/nimblepkg/options.nim

```diff
@@ -305,7 +305,7 @@ proc parseFlag*(flag, val: string, result: var Options, kind = cmdLongOption) =
 proc initOptions*(): Options =
   result.action.typ = actionNil
   result.pkgInfoCache = newTable[string, PackageInfo]()
-  result.nimbleDir = ""
+  result.nimbleDir = getEnv("NIMBLE_DIR")
```

Comment:
Please output a warning or a hint message that this environment variable is being used.

Reply:
I can do that.  I changed it here so that the command line option parsed later can override this environment variable, and when the command line option takes precedence, I guess we need another hint to disambiguate nimble's decision.  Do you think it's okay to do that, or would it be too chatty?

Reply:
It will be best if a message is only printed when the environment variable is used. So maybe instead of setting it here you could check whether ``nimbleDir`` is empty later down the line?